source/Octopus.Tentacle.Client/Scripts/IScriptExecutor.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Octopus.Tentacle.Client.EventDriven;
+using Octopus.Tentacle.Client.Scripts.Models;
+using Octopus.Tentacle.Contracts;
+
+namespace Octopus.Tentacle.Client.Scripts
+{
+    public interface IScriptExecutor 
+    {
+        Task<ScriptOperationExecutionResult> StartScript(ExecuteScriptCommand command,
```

Comment:
It might make sense to document that the resulting CommandContext should be used for the next call.

Reply:
Done